Classes/WPEditorToolbarButton.m

```diff
@@ -1,19 +1,25 @@
 #import "WPEditorToolbarButton.h"
+#import <WordPress-iOS-Shared/WPStyleGuide.h>
+
+static NSString* const CircleLayerKey = @"circleLayer";
+static int TouchAnimationCircleRadius = 15;
+static int TouchAnimationCircleRadiusiPad = 17;
+static CGFloat TouchAnimationDuration = 0.4f;
+static CGFloat TouchAnimationInitialOpacity = 0.8f;
+
+static CGFloat AnimationDurationNormal = 0.3f;
+static CGFloat HighlightedAlpha = 0.1f;
+static CGFloat NormalAlpha = 1.0f;
```

Comment:
For the `CGFloats`, Its safe to drop the `f` and let the compiler worry about exactly which flavor of floating point the `CGFloat` should actually be.